pre_commit_hooks/xml_format.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+
+from lxml import etree
```

Comment:
My computer seems to be doing something weird with lxml or one of it's dependencies. It works fine but it outputs a `default.profraw` file whenever I use python in an environment with it :/
I'll try to dig a bit deeper to see what's going on.

Reply:
That looks like an LLVM profiling file extension (`.profraw`). Can you try it in a second, clean environment?

Also, where'd you get `lxml` from? pip, conda-forge, conda-defaults?

Reply:
That's bizarre. When used through `pre-commit` it will be run a tiny `virtualenv` populated by `pip`, so hopefully that path is clean... it will need to be, because if a file is created during a `pre-commit` step then the `pre-commit` will stop you.

Reply:
Got `lxml` from conda-forge but yeah I tried with a clean `conda create -n test2 python` env and it too gives me the a `default.profraw` file without `lxml` installed :/

Reply:
Hmm, how about any strange environment settings? Any recent changes to your bashrc etc?

Reply:
I don't think so, looking more into what would use LLVM.

Reply:
There's probably a weird python build with the wrong compiler flags set somewhere...